tests/test_datetime_defaults.py

```diff
@@ -1,29 +1,44 @@
 # -*- coding: utf-8 -*-
 from datetime import date, datetime
+
+import pytest
 import sqlalchemy as sa
 
 from sqlalchemy_defaults import Column
-from tests import TestCase
 
+from . import MYSQLD_VERSION
+
+
+@pytest.fixture
+def User(Base, lazy_options):
+
+    class User(Base):
+        __tablename__ = 'user'
+        __lazy_options__ = lazy_options
+
+        id = Column(sa.Integer, primary_key=True)
+        created_at = Column(sa.DateTime, auto_now=True)
+        fav_day = Column(
+            sa.Date, min=date(2000, 1, 1), max=date(2099, 1, 1)
+        )
+
+    return User
 
-class TestDateTimeDefaults(TestCase):
-    column_options = {}
 
-    def create_models(self, **options):
-        class User(self.Model):
-            __tablename__ = 'user'
-            __lazy_options__ = options
+@pytest.fixture
+def models(User):
+    return [User]
 
-            id = Column(sa.Integer, primary_key=True)
-            created_at = Column(sa.DateTime, auto_now=True)
-            fav_day = Column(
-                sa.Date, min=date(2000, 1, 1), max=date(2099, 1, 1)
-            )
 
-        self.User = User
+@pytest.mark.usefixtures('lazy_configured', 'Session')
+class TestDateTimeDefaults(object):
 
-    def test_autonow(self):
-        column = self.User.created_at
+    @pytest.mark.skipif(
```

Comment:
Not using `skipif` AFAIK. Couldn't find anything about it in [its documentation](https://pytest.org/latest/skipping.html).